lib/ansible/modules/cloud/amazon/ec2_transit_gateway_info.py

```diff
@@ -0,0 +1,265 @@
+                        side  of  a  BGP session. The range is 64512 to 65534 for
+                        16-bit ASNs and 4200000000 to 4294967294 for 32-bit ASNs.
+                    returned: always
+                    type: str
+                    sample: 64512
+                 auto_accept_shared_attachments:
+                    description:
+                       - Indicates whether attachment requests  are  automatically accepted.
+                    returned: always
+                    type: str
+                    sample: "enable"
+                 default_route_table_association:
+                    description:
+                      - Indicates  whether resource attachments are automatically
+                        associated with the default association route table.
+                    returned: always
+                    type: str
+                    sample: "disable"
+                 association_default_route_table_id:
+                    description:
+                      - The ID of the default association route table.
+                    returned: when present
+                    type: str
+                    sample: "rtb-11223344"
+                 default_route_table_propagation:
+                    description:
+                      - Indicates  whether  resource  attachments   automatically
+                        propagate routes to the default propagation route table.
+                    returned: always
+                    type: str
+                    sample: "disable"
+                 dns_support:
+                    description:
+                      - Indicates whether DNS support is enabled.
+                    returned: always
+                    type: str
+                    sample: "enable"
+                 propagation_default_route_table_id:
+                    description:
+                      - The ID of the default propagation route table.
+                    returned: when present
+                    type: str
+                    sample: "rtb-11223344"
+                 vpn_ecmp_support:
+                    description:
+                      - Indicates  whether  Equal Cost Multipath Protocol support
+                        is enabled.
+                    returned: always
+                    type: str
+                    sample: "enable"
+        owner_id:
+            description: The AWS account number ID which owns the transit gateway.
+            returned: always
+            type: str
+            sample: "1234567654323"
+        state:
+            description: The state of the transit gateway.
+            returned: always
+            type: str
+            sample: "available"
+        tags:
+            description: A dict of tags associated with the transit gateway.
+            returned: always
+            type: dict
+            sample:
+              Name: "A sample TGW"
```

Comment:
Shouldn't this be
```suggestion
              '{"Name": "A sample TGW"}
```
?

Reply:
wrapped sample in proper structure.